Provide getmntinfo and MNT_* flags needed by trash >= 2.0.0

Needed until https://github.com/rust-lang/libc/pull/2907 lands
in a libc release.

Index: modcargo-crates/libc-0.2.132/src/unix/bsd/netbsdlike/openbsd/mod.rs
--- modcargo-crates/libc-0.2.132/src/unix/bsd/netbsdlike/openbsd/mod.rs.orig
+++ modcargo-crates/libc-0.2.132/src/unix/bsd/netbsdlike/openbsd/mod.rs
@@ -1612,6 +1612,15 @@ pub const PTRACE_FORK: ::c_int = 0x0002;
 
 pub const WCONTINUED: ::c_int = 8;
 
+// https://github.com/openbsd/src/blob/master/sys/sys/mount.h#L416
+
+/// Synchronously wait for I/O to complete.
+pub const MNT_WAIT: ::c_int = 1;
+/// Start all I/O, but do not wait for it.
+pub const MNT_NOWAIT: ::c_int = 2;
+/// Push data not written by filesystem syncer.
+pub const MNT_LAZY: ::c_int = 3;
+
 // search.h
 pub const FIND: ::ACTION = 0;
 pub const ENTER: ::ACTION = 1;
@@ -1870,6 +1879,7 @@ cfg_if! {
             // these functions use statfs which uses the union mount_info:
             pub fn statfs(path: *const ::c_char, buf: *mut statfs) -> ::c_int;
             pub fn fstatfs(fd: ::c_int, buf: *mut statfs) -> ::c_int;
+            pub fn getmntinfo(mntbufp: *mut *mut ::statfs, mode: ::c_int) -> ::c_int;
         }
     }
 }
